bsp/src/mill/bsp/BSP.scala

```diff
@@ -23,12 +23,21 @@ object BSP extends ExternalModule with CoursierModule with BspServerStarter {
     Agg(Dep.parse(BuildInfo.millBspWorkerDep))
   }
 
-  private def bspWorkerLibs: T[Agg[PathRef]] = T {
-    mill.modules.Util.millProjectModule(
-      "MILL_BSP_WORKER",
-      "mill-bsp-worker",
-      repositoriesTask()
-    )
+  private def bspWorkerLibs: T[Agg[PathRef]] = {
+    sys.props.getOrElse("MILL_UNDER_TEST", "0") match {
```

Comment:
What's this change for? AFAICT, it doesn't modify the security characteristics at all: the existing `millProjectModule` already uses `sys.props` to control things, so anyone who has control over the `MILL_BSP_WORKER` property would also have control over the `MILL_UNDER_TEST` property

Reply:
Oh, this seems to be left over. I searched for the issues which later turned out to come from #2419. I'll revert it.